src/__tests__/createActions-test.js

```diff
@@ -0,0 +1,29 @@
+import { createActions } from '../';
+
+describe('createActions()', () => {
+  describe('resulting action creator map', () => {
+    const types = {
+      ACTION: 'ACTION',
+      MY_LONG_ACTION: 'MY_LONG_ACTION'
+    };
+
+    it(`apply a default string transformation function to action types
+        when mapping to action creators`, () => {
+      const actionCreators = createActions(types);
+
+      expect(actionCreators).to.have.property('action');
+      expect(actionCreators).to.have.property('myLongAction');
+    });
```

Comment:
If I pass in an action `fooBar`, the behavior will be to make the action creator key `foobar`.

```js
function camelize(s) {
  return s.toLowerCase().replace(/_\w/g, x => x[1].toUpperCase())
}

console.log(camelize('fooBar')) // foobar
```

Is this the intended behavior?